bats/tests/k8s/helm-install-rancher.bats

```diff
@@ -29,9 +105,10 @@ deploy_rancher() {
     local host
     host=$(traefik_hostname) || return
 
+    trace "Installing rancher $rancher_chart_version"
```

Comment:
I think this is important enough information that I would like to see this always printed:

```suggestion
    echo "# Installing rancher $rancher_chart_version" >&3
```

So we get output like this:

```
 ✓ wait_for_traefik 1.27.1
 ✓ deploy_rancher 1.27.1
   Installing rancher 2.8.0
 ✓ verify_rancher 1.27.1
```

Reply:
Maybe we should have a `comment` function for this, similar to `trace`?